src/oge/output_data.py

```diff
@@ -287,85 +287,59 @@ def output_data_quality_metrics(
         )
 
 
-def output_plant_data(
-    df: pd.DataFrame,
+def write_plant_data_to_results(
```

Comment:
Updated to specifically output monthly and annual plant level data, or output monthly or annual subplant-level data